From 3c21061a69dcac6260a1b7ec599fc4f5543a43d6 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 16 Feb 2024 09:42:51 +0100
Subject: [PATCH 0673/1695] dts: stm32: enable SAES on stm32mp257f-dk

Set SAES status to "okay" in stm32mp257f-dk.dts

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2e48ae0a6ae74407ae2e4be218019f0ac31a2cce dts: stm32: enable SAES on stm32mp257f-dk]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: Ie0de50797d63e0997de640cd33cd05c07b899544
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/360598
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index f819e8b0f..e03e01594 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -383,6 +383,10 @@
 	status = "okay";
 };
 
+&saes {
+	status = "okay";
+};
+
 &scmi_regu {
 	voltd-vddcore {
 		reg = <VOLTD_SCMI_STPMIC2_BUCK2>;
-- 
2.47.2

